app/models/concerns/sufia/ability.rb

```diff
@@ -52,6 +52,7 @@ def editor_abilities
 
       can :create, TinymceAsset
       can [:create, :update], ContentBlock
+      can :edit, ::SolrDocument
```

Comment:
@jcoyne yeah, I've used that approach before, however, I think I prefer this. You don't have to maintain another group, i.e. visibility's "registered" or "public", and that's one less Fedora resource created per object. Plus, if the object looses that admin group somehow, you wouldn't be able to edit it.

OTOH, using rights metadata would also push the admin ability to the Fedora level, and enable it there once the Fedora ACL/Hydra ACL alignment is complete... (if/when that happens)